src/service/user.ts

```diff
@@ -1,49 +1,125 @@
-import { PrismaClient } from "@prisma/client";
+import { Prisma, PrismaClient, School } from "@prisma/client";
```

Comment:
불필요한 import를 지웠어요 :D
```suggestion
import { PrismaClient } from "@prisma/client";
```